analytics/policy/README.md

```diff
@@ -1 +1,105 @@
-# README about policy module
+# Policy Evaluation Module
+
+## Description
+
+To determine illegal rental listings on Airbnb, it is important to make sure business registry for each listing is validated. A system composed of policies and rules needs to be set up to effectively handle a large number of listings. 
+
+## Package Management
+
+This package uses [Poetry](https://python-poetry.org/) to manage dependencies and
+isolated [Python virtual environments](https://docs.python.org/3/library/venv.html).
+
+To proceed,
+[install Poetry globally](https://python-poetry.org/docs/#installation)
+onto your system.
+
+### Dependencies
+
+Dependencies are defined in [`pyproject.toml`](./pyproject.toml) and specific versions are locked
+into [`poetry.lock`](./poetry.lock). This allows for exact reproducible environments across
+all machines that use the project, both during development and in production.
+
+To install all dependencies into an isolated virtual environment:
+
+> Append `--sync` to uninstall dependencies that are no longer in use from the virtual environment.
+
+```bash
+$ poetry install
```

Comment:
i love the fact that you are trying to have a doc that help devs that setup their environment. 
consider using docker to speed up the setup so that the devs can simply use docker and have everything being the same.
i.e. python version, dependencies. etc. all in a single command. 
that way when you deploy it will be simple as well.

Reply:
Do you think there should be docker for each module or the whole project?
Also, we dont have too complex stuff and many devs yet so is it ok to set it up later? Just wanna know the tradeoff, i think i can set it up :)

Reply:
and github actions for ci too?

Reply:
its okay to setup later, just a comment to keep in mind

Reply:
i will create a task in backlog in case we need to this